python/stubs/ncs/dp.pyi

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Callable, Dict, Optional, Union, TypeVar
+import _ncs
+import ncs
+from ncs.maagic import Node
+from ncs.maapi import Transaction
+
+class Daemon:
+    class State: ...
+
+
+ActionObj = TypeVar('ActionObj', bound='Action')
+
+ACallback6 = Callable[[ActionObj, _ncs.UserInfo, str, _ncs.HKeypathRef, Node, Node],
```

Comment:
doubly indexed typing here is a bit confusing - isn't `ACallback6` ActionCallback[Input/Params]6` in reality?

Reply:
sorry, that was incorrect reading of multi-line typedef for Callable on my side... it indeed has correct form of `Callable[[a, b, ...], res]`